source/features/pr-easy-toggle-files.tsx

```diff
@@ -0,0 +1,48 @@
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+const selectorHeaderBar = '.file-header.file-header--expandable';
+
+function toggleFile(event: delegate.Event<Event, HTMLElement>): void {
+	const elementClicked = event.target as HTMLElement;
+	const {parentElement} = elementClicked;
+
+	if (parentElement === null) {
+		return;
+	}
```

Comment:
This never happens. If you're trying to fix the TypeScript error use `!` after the variable name.

Reply:
Indeed, I used it for other parts for forgot for this one.